src/sonic-yang-models/yang-models/sonic-interface.yang

```diff
@@ -57,6 +57,12 @@ module sonic-interface {
 						path "/vrf:sonic-vrf/vrf:VRF/vrf:VRF_LIST/vrf:name";
 					}
 				}
+
+				leaf mpls {
+					type string {
+						pattern "enable|disable";
+					}
```

Comment:
need description here, what does enable/disable mean here.

Reply:
@lguohan description is added.